analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -168,9 +168,15 @@ open class Npm(
         }
 
         return declaredLicenses.mapTo(sortedSetOf()) { declaredLicense ->
-            // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
-            // private or unpublished package under any terms", which corresponds to SPDX's "NONE".
-            if (declaredLicense == "UNLICENSED") SpdxLicense.NONE else declaredLicense
+            when {
+                // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
+                // private or unpublished package under any terms", which corresponds to SPDX's "NONE".
+                declaredLicense == "UNLICENSED" -> SpdxLicense.NONE
+                // NPM allows to declare non-SPDX licenses only by referencing a license file. Avoid reporting an
+                // [OrtIssue] by mapping this to a valid license identifier.
+                declaredLicense.startsWith("SEE LICENSE IN ") -> "LicenseRef-scancode-unknown-license-reference"
```

Comment:
IMO we should stay scanner-agnostic here and not use a LicenseRef from the `scancode` namespace here, but from our own `ort` namespace, i.e. `LicenseRef-ort-unknown-license-reference` or similar.

Reply:
I totally agree. However, looking at `DeclaredLicenseMapping` there are multiple mappings to `ScanCode` licenses and thus this change is consistent with that. I propose we create a ticket for removing ScanCode license identifiers from the analyzer consistently. Would you be fine with that approach?

Reply:
Yes, there are some such mappings in `SpdxDeclaredLicenseMapping` of the `spdx-utils` module that use the `licenseRef()` helper function. I agree these need to be fixed separately (and ASAP), but I don't see why that should justify to now introduce code to the `analyzer` module which we already know to be wrong. So I'd prefer to do this change right from the start.

I would have agreed with you if you had introduced another mapping to `SpdxDeclaredLicenseMapping` in `spdx-utils`, but this is something different to me.

Reply:
Done, even though I'm not really convinced.